### DISCUSSION TITLE
Can this also be used to slow down the refresh rate?

### DISCUSSION THREAD

**Discussion by Anish-Sek** [derekantrican/GAS-ICS-Sync#355] [General]
Say I want it to update once a week, can I just do 60*24*7 as the minutes?

**Comment by jonas0b1011001** [comment]
You would have to change [code.gs line 105](https://github.com/derekantrican/GAS-ICS-Sync/blob/e5cd8e02417ee38a04eadbd795201822dbac2e2d/Code.gs#L105) to
`ScriptApp.newTrigger("startSync").timeBased().everyWeeks(howFrequent).create();`

And run the script with `howFrequent = 1;`

**Reply by Anish-Sek** [reply]
Hey, thank you for your quick response! When I try doing that, it says Exception: You tried to create a weekly trigger, but did not specify which day in the week.

**Reply by Anish-Sek** [reply]
Nvm I got it used .onWeekDay(ScriptApp.WeekDay.MONDAY)

**Comment by Lonestarjeepin** [comment]
I submitted two different pull requests, #358 & #372 that could accomplish this as well.  #358 can get you an actual trigger as infrequent as once per day (1,440 minutes), but you could use #372 to simply input 10,080 as the SyncDelay on a given calendar.  Then, regardless of your trigger frequency, that particular calendar would only refresh once per week.
